File: product-processor-service/src/main/java/com/example/product_processor_service/service/product/grpc/client/ProductGrpcClientServiceManager.java
package com.example.product_processor_service.service.product.grpc.client;

import com.example.grpc.user.ProductProtoServiceGrpc;
import org.springframework.stereotype.Service;

@Service
public class ProductGrpcClientServiceManager extends ProductGrpcClientService {
    public ProductGrpcClientServiceManager(ProductProtoServiceGrpc.ProductProtoServiceBlockingStub stub) {
        super(stub);
    }
}

File: product-manager-service/src/main/kotlin/com/example/product_manager_service/service/product/grpc/ProductGrpcClientServiceManager.kt
package com.example.product_manager_service.service.product.grpc

import com.example.grpc.user.ProductProtoServiceGrpc
import org.springframework.stereotype.Service

@Service
class ProductGrpcClientServiceManager(stub: ProductProtoServiceGrpc.ProductProtoServiceBlockingStub) : ProductGrpcClientService(stub) {
}